1904-11-24.xml

```diff
@@ -0,0 +1,6922 @@
+                            <cell>"</cell>
+                            <cell>
+                                <measure unit="£">8 11/16</measure></cell>
+                            <cell>"</cell>
+                            <cell><measure unit="£">—8/4</measure></cell>
+                        </row>
+                        <row>
+                            <cell> The Nile Land</cell>
+                            <cell>"</cell>
+                            <cell><measure unit="£">9 –</measure>
+                            </cell>
+                            <cell>"</cell>
+                            <cell><measure unit="£">10— </measure></cell>
+                        </row>
+                        <row>
+                            <cell>Sucr ris ot Rafficerie d'Egypte</cell>
+                            <cell>Fcs.</cell>
+                            <cell>202 –– </cell>
+                            <cell>a</cell>
+                            <cell><measure unit="£">208— </measure></cell>
+                        </row>
+                    </table>
+                </div>
+                <div type="item" xml:id="deg-el-rrat01">
+                    <head>ALEXANDRIA &amp; RAMLEH RAILWAY Co.</head>
+                    <table cols="4" xml:id="deg-ta-rrat01">
+                        <head>RECETTES</head>
+                        <row>
+                            <cell cols="4">du <date when="1905-07-02">Dimanche 18 Nov.</date>
+                                    aun<date when="1905-07-08">Samedi 19 Nov. 1904</date></cell>
+                        </row>
+                        <row role="label">
+                            <cell/>
+                            <cell>Billets</cell>
+                            <cell>Abonnements</cell>
+                            <cell>Carnets et Divers</cell>
+                        </row>
+                        <row>
+                            <cell/>
+                            <cell>P.T.</cell>
+                            <cell>P.T.</cell>
+                            <cell>P.T.</cell>
+                        </row>
+                        <row>
+                            <cell>Année courante</cell>
+                            <cell><measure unit="pt">57,257</measure></cell>
+                            <cell><measure unit="pt">8,869</measure></cell>
+                            <cell><measure unit="pt">8,839</measure></cell>
+                        </row>
+                        <row>
+                            <cell>,, derniere</cell>
+                            <cell><measure unit="pt">50,921</measure></cell>
+                            <cell><measure unit="pt">1,845</measure></cell>
+                            <cell><measure unit="pt">1,021</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Augmentation</cell>
+                            <cell><measure unit="pt">6,846</mea
```

Comment:
move this `</div>` tag to the end of the prepaid advertisements section, so that all the ad items are contained in it.